src/components/Decks/components/DecksComponent.tsx

```diff
@@ -1,16 +1,39 @@
-import { Decks } from 'models/Deck'
+import { Deck, DeckName, Decks } from 'models/Deck'
 import './DecksComponent.scss'
+import { useState } from 'react'
 
 interface Props {
 	decks: Decks
 }
 
 const DecksComponent: React.FC<Props> = (props) => {
+	const [selected, setSelected] = useState<DeckName>('hiragana')
+	const active = 'active'
+	const desactive = 'desactive'
+
 	const { decks } = props
 	console.log('decks =>', decks)
+	const onClickDecks = (deckRecived: DeckName) => {
+		setSelected(deckRecived)
+	}
+
 	return (
-		<div>
-			<h1>Deck</h1>
+		<div className='decks'>
+			<div
+				className={`deck ${selected === 'hiragana' ? 'selected' : ''}`}
```

Comment:
Apply here the same constants you've to declare based on the comment before

Reply:
Done